cpp/src/fil/treelite_import.cu

```diff
@@ -18,7 +18,7 @@
  * fil.cu can make a `forest` object out of it. */
 
 #include "common.cuh"    // for num_trees, tree_num_nodes
-#include "internal.cuh"  // for MAX_PRECISE_INT_FLOAT, BITS_PER_BYTE, cat_feature_counters, cat_sets, cat_sets_owner, categorical_sets, leaf_algo_t
+#include "internal.cuh"  // for MAX_FIL_INT_FLOAT, BITS_PER_BYTE, cat_feature_counters, cat_sets, cat_sets_owner, categorical_sets, leaf_algo_t
```

Comment:
I appreciate these comments **so** much. Thank you for keeping them up-to-date! :heart: